src/DotNetWorker/Http/HttpRequestDataExtensions.cs

```diff
@@ -19,7 +19,7 @@ public static class HttpRequestDataExtensions
         /// </summary>
         /// <param name="request">The request from which to read.</param>
         /// <param name="encoding">The encoding to use when reading the string. Defaults to UTF-8</param>
-        /// <returns>A <see cref="Task{string?}"/> that represents the asynchronous read operation.</returns>
+        /// <returns>A task that represents the asynchronous read operation.</returns>
```

Comment:
We should be able to have a generic type reference here, correct?

Reply:
I think you can do Task<T> -- but not Task<specifictype> -- I checked some other official docs and they just say "A task..." -- but let me look for other examples.

Reply:
Was this maybe just an issue with using the alias? (`string` vs `String`)?

Reply:
oh... it was the `?` throwing it off. fixed it up.